marketplacetests/marketplace/regions/debug.py

```diff
@@ -21,6 +21,8 @@ def tap_back(self):
         self.marionette.find_element(*self._back_button_locator).tap()
 
     def select_region(self, region):
-        self.marionette.find_element(*self._region_select_locator).tap()
+        element = self.marionette.find_element(*self._region_select_locator)
+        self.marionette.execute_script('arguments[0].scrollIntoView(false);', [element])
```

Comment:
Good idea. Should we be even more explicit though? That comment just mentions a bug, but not the fact that we won't need this workaround in later gaia versions. So maybe the comment should mention that. What do you think?